packages/firebase_core/firebase_core_web/lib/src/firebase_core_web.dart

```diff
@@ -32,7 +32,7 @@ class FirebaseCoreWeb extends FirebasePlatform {
   }) async {
     firebase.App? app;
 
-    if (name == defaultFirebaseAppName) {
+    if (name == defaultFirebaseAppName && !kIsWeb) {
```

Comment:
Is this needed at all now? I thought the latest merge from @russellwheatley removed this

Reply:
Russell didn't merge it with master, and I'm basing this PR with master

Reply:
Sorry, when I said merged with master, I meant on this PR: https://github.com/FirebaseExtended/flutterfire/pull/6549